_layouts/syllabus.html

```diff
@@ -55,6 +55,11 @@
       <a href="lectures/{{ page.slidesName }}.pdf">[pdf]</a>
       </p>
     {% endif %}
+    {% for oh_plus_plus in page.ohPlusPlus %}
+      <p>
+      <a href="/static/w17/oh-plus-plus/{{ oh_plus_plus }}">OH++ Slides</a>
```

Comment:
That link isn't meant to be correct. Replace SEMESTER with `w17` and `XX` with `5`, for example. When we say "GitHub rendering", we mean "link to the file on github so it looks nicer than a vanilla markdown file.